Readme.md

```diff
@@ -1,56 +1,16 @@
-# CaPTk:  Cancer Imaging Phenomics Toolkit 
+# Binaries
```

Comment:
This needs to change, please use the original Readme.md. The information about the binaries (applications and Qt) needs to go into a separate file under ```binaries```

Reply:
Resolved in my fork.